tools/toolchain/dbuild

```diff
@@ -16,6 +16,9 @@ EOF
     exit 1
 }
 
+if env | grep DOCKER_USERNAME > /dev/null ; then
+  echo $DOCKER_PASSWORD | docker login --username=$DOCKER_USERNAME --password-stdin
+fi
```

Comment:
Docker/podman credentials can be stored elsewhere. e.g. ~/.docker/config.json. Those don't need any changes to dbuild, they just happen natually.

Reply:
i.e. this should be done by the user, even if the user is Jenkins.